plugins/module_utils/common/api/base_paths.py

```diff
@@ -0,0 +1,323 @@
+# Copyright (c) 2025 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Centralized base paths for DCNM/NDFC API endpoints.
```

Comment:
@mtarking and I have started to standardize around using `ND` instead of `DCNM/NDFC` for any comments, doc headers and other documentation.  Not suggesting you modify the code.